src/storage/src/hummock/compactor/shared_buffer_compact.rs

```diff
@@ -460,6 +462,11 @@ fn generate_splits(
                 }
             }
         }
+        for (table_id, sz) in table_size {
```

Comment:
We should unified the split logic between `flush` and `compaction`

Reply:
Emm, the only problem is that compute-node can not know the configuration of compaction group